docs/user/gen-docs/01-170-kyma_alpha_reference-instance.md

```diff
@@ -0,0 +1,37 @@
+# kyma alpha reference-instance
+
+Add an instance reference to a shared service instance
+
+## Synopsis
+
+Use this command to add an instance reference to a shared service instance on the SAP Kyma platform
```

Comment:
```suggestion
Use this command to add an instance reference to a shared service instance on SAP BTP.
```

Reply:
This command is adding things the kyma cluster so I would suggest adding the `on the Kyma cluster` instead of `on SAP BTP` at the end of the description